### PR TITLE
update BART example

### PR DESCRIPTION
**Pull request by aloctavodia**
This mainly reflects changes in the new API.

### CODE REVIEW

File: myst_nbs/case_studies/BART_introduction.myst.md [hunk 1]
```diff
@@ -33,8 +33,8 @@ print(f"Running on PyMC v{pm.__version__}")
 ```
 
 ```{code-cell} ipython3
-RANDOM_SEED = 8457
-rng = np.random.RandomState(RANDOM_SEED)
+RANDOM_SEED = 5781
+np.random.seed(RANDOM_SEED)
```
Comment (OriolAbril) [inline myst_nbs/case_studies/BART_introduction.myst.md:37]: I think the global seed doesn't have any effect on v4? if the rng is not used then we can only define the random seed variable to pass to `pymc.sample`

Comment (aloctavodia) [reply myst_nbs/case_studies/BART_introduction.myst.md:37]: I double-checked and it has an effect on BART.

Comment (ricardoV94) [reply myst_nbs/case_studies/BART_introduction.myst.md:37]: It shouldn't ;)

Comment (aloctavodia) [reply myst_nbs/case_studies/BART_introduction.myst.md:37]: It is not clear to me what I have to change on pymc-bart, but I will give it a try. Any suggestion is welcome.

**Review by OriolAbril — APPROVED** [review]

**Pull request merged**